storage/store/cache.go

```diff
@@ -19,6 +19,8 @@ func withLimit[K comparable, V any](limit uint) func(*Cache[K, V]) {
 
 type storeFunc[K comparable, V any] func(rw storage.ReaderBatchWriter, key K, val V) error
 
+const DefaultCacheSize = uint(1000)
```

Comment:
Why not put this definition closer to the usage. In the `storage/store/chunks_queue.go` file.